flocker/control/test/test_persistence.py

```diff
@@ -479,7 +431,7 @@ def _build_node(applications):
     APPLICATIONS,
     # If we add this hint on the number of applications, Hypothesis is able to
     # run many more tests.
-    average_size=10,
+    average_size=3,
```

Comment:
The short text thing sped up tests from 35 seconds to 26 seconds. Unfortunately applying it would involve another Buildbot CI round which means I'm not going to be able to commit this PR today, so I'm not going to bother.